From deb79196fd8f7696be86406104a0e0c0416c5acb Mon Sep 17 00:00:00 2001
From: Virendra Pathak <vpathak@broadcom.com>
Date: Wed, 10 Dec 2014 14:34:07 +0530
Subject: i2c: multinode fixes

Update multinode default DTS to add eeprom entry for xlp8xx and the
temperature sensor entry for xlp9xx
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/arch/mips/boot/dts/xlp8xx-linux-mn-kernel.dts b/arch/mips/boot/dts/xlp8xx-linux-mn-kernel.dts
index 788cd83..69b1f0e 100644
--- a/arch/mips/boot/dts/xlp8xx-linux-mn-kernel.dts
+++ b/arch/mips/boot/dts/xlp8xx-linux-mn-kernel.dts
@@ -104,6 +104,10 @@
 				compatible = "national,lm90";
 				reg = <0x4c>;
 			};
+			eeprom@57 {
+				compatible = "xlpge_eeprom";
+				reg = <0x57>;
+			};
 		};
 
 		nor_flash@1,0 {
diff --git a/arch/mips/boot/dts/xlp9xx-linux-mn-kernel.dts b/arch/mips/boot/dts/xlp9xx-linux-mn-kernel.dts
index 4953d85..6db78b3 100644
--- a/arch/mips/boot/dts/xlp9xx-linux-mn-kernel.dts
+++ b/arch/mips/boot/dts/xlp9xx-linux-mn-kernel.dts
@@ -70,6 +70,11 @@
 				reg = <0x68>;
 			};
 
+			dtt@2c {
+                                compatible = "max6653";
+                                reg = <0x2c>;
+			};
+
 			eeprom@57 {
                                 compatible = "xlpge_eeprom";
                                 reg = <0x57>;
-- 
1.7.1

